juriscraper/opinions/united_states/state/nytrial.py

```diff
@@ -114,110 +115,56 @@ def _download_backwards(self, target_date: date) -> None:
         """
         self.url = self.build_url(target_date)
 
-    def _make_html_tree(self, text: str) -> HtmlElement:
-        """Overrides method from AbstractSite
-        Taken from nyappterm_1st that implements this same site
-        but needs different parsing
-
-        :param text: html as text
-        :returns: html object
-        """
-        return get_html5_parsed_text(text)
-
     def extract_from_text(self, scraped_text: str) -> Dict[str, Any]:
-        """Extracts docket number, judge name and citation, if available
+        """Extract values from opinion's text
 
-        :param scraped_text: can be cleansed HTML with some tags
-                             or plain PDF text, processed by courtlistener
-        :return: Dictionary where each key is a courtlistener Django Model name
-                and each value is dictionary of model's fields
+        Limit to first 3000 characters to avoid picking a citation or
+        docket number from opinion's argumentation. All values we are parsing
+        appear on summary tables at the beginning of the document, and
+        sometimes may not exist
```

Comment:
is this correct.  I think the regex picks up judge author names and docket numbers after the tables.

Reply:
I phrased it badly. You are right, we do pick up data from a few lines after the second table